### DISCUSSION TITLE
Unable to select subset of columns

### DISCUSSION THREAD

**Discussion by eduardomanuel** [opensource9ja/danfojs#464] [Q&A] [answered]
I have a Data Frame with a bunch of columns.  I need to create a new Data Frame with just a subset of the columns from the orig Data Frame.  Whenever i try to do that, i only get the last column specified.   

I specify the subset like :  my_df["col_1","col2","col_n"] ... I end up  with a new Data Frame with just col_n

Is that a known bug or am i doing something wrong?

**Comment by risenW** [comment, marked as answer]
> I have a Data Frame with a bunch of columns. I need to create a new Data Frame with just a subset of the columns from the orig Data Frame. Whenever i try to do that, i only get the last column specified.
> 
> I specify the subset like : my_df["col_1","col2","col_n"] ... I end up with a new Data Frame with just col_n
> 
> Is that a known bug or am i doing something wrong?

You should use the loc function instead. See https://danfo.jsdata.org/api-reference/dataframe/danfo.dataframe.loc

**Reply by eduardomanuel** [reply]
Thank you!   That worked perfectly! 👌
